### DISCUSSION TITLE
is this a bug?

### DISCUSSION THREAD

**Discussion by jonas-frey** [draivin/hsnips#175] [General]
I have the snippet

```
snippet rem: bA
**Remark:**
endsnippet
```
in my `markdown.hsnips` file, and it replaces `rem:` at the start of a line by `**Remark:**` as expected, but it also replaces `theorem:`, which I did *not* expect and was very confused about. Is this a bug?

**Comment by FluffLescure** [comment]
> as expected, but it also replaces `theorem:`

Where does `theorem:` come from? it is generic text in the markdown? is it a second snippet? where was it placed?

I've tested the snippet and there isn't anything being removed outside its expected function.
